platforms/android/tangram/src/main/java/com/mapzen/tangram/MapController.java

```diff
@@ -1134,4 +1138,30 @@ String getFontFallbackFilePath(int importance, int weightHint) {
         return fontFileParser.getFontFallback(importance, weightHint);
     }
 
+    // Tmp Asset Path
+    // ==============
+    String copyAssetToTmpPath(String path) throws IOException {
+        String tmpDir = mapView.getContext().getExternalCacheDir().getAbsolutePath();
+        InputStream in = assetManager.open(path);
```

Comment:
We should add a `try`, `catch` block here with the right exception check (`IOException`), return either an empty string or null string and do the appropriate error check within tangram core in that case.